Formula/wrk.rb

```diff
@@ -3,6 +3,14 @@ class Wrk < Formula
   homepage "https://github.com/wg/wrk"
   url "https://github.com/wg/wrk/archive/4.2.0.tar.gz"
   sha256 "e255f696bff6e329f5d19091da6b06164b8d59d62cb9e673625bdcd27fe7bdad"
+  # License is modified Apache 2.0 with addition to Section 4 Redistribution:
+  #
+  # (e) If the Derivative Work includes substantial changes to features
+  #     or functionality of the Work, then you must remove the name of
+  #     the Work, and any derivation thereof, from all copies that you
+  #     distribute, whether in Source or Object form, except as required
+  #     in copyright, patent, trademark, and attribution notices.
+  license "Apache-2.0"
```

Comment:
Need confirmation if modified Apache-2.0 can be licensed as Apache-2.0.

A couple repositories have labeled it Apache-2.0 (e.g. Alpine, MacPorts, and NixOS). Debian refers to it as `Apache-2.0~glozer`

Reply:
I'd say this is a `:cannot_represent` again.